scan_mask_node:
  ros__parameters:

    input_topic: /scan
    output_topic: /scan_masked

# sectors are angles in radians, 0 is front of robot, direction is against the clock
    sectors: [
      1.31, 1.83,
      2.77, 2.92,
      3.36, 3.51,
      4.45, 4.97
      ]



